test/flux_calculator_tests.jl

```diff
@@ -44,38 +45,21 @@ function atmos_turbulent_fluxes!(sim::DummySimulation, csf)
 end
 
 # Test for the PartitionedStateFluxes() case
-# parameter set generated CLIMAParamerers and helper functions from SurfaceFluxes.jl
-function create_uf_parameters(toml_dict, ::UF.BusingerType)
-    FT = CP.float_type(toml_dict)
-    aliases = ["Pr_0_Businger", "a_m_Businger", "a_h_Businger", "ζ_a_Businger", "γ_Businger"]
-
-    pairs = CP.get_parameter_values!(toml_dict, aliases, "UniversalFunctions")
-    pairs = (; pairs...) # convert to NamedTuple
-
-    pairs = (;
-        Pr_0 = pairs.Pr_0_Businger,
-        a_m = pairs.a_m_Businger,
-        a_h = pairs.a_h_Businger,
-        ζ_a = pairs.ζ_a_Businger,
-        γ = pairs.γ_Businger,
-    )
-    return UF.BusingerParams{FT}(; pairs...)
-end
-
-function create_parameters(toml_dict, ufpt)
+# Create parameters using CLIMAParameters.jl
+function create_parameters(toml_dict, ::UF.BusingerType)
```

Comment:
What are the param wrapper functions used for? `create_parameters`, `get_thermo_params`, etc
Each package should now own its own parameter constructor which takes in a FT or a toml dict, so you shouldn't need to make your own name map or constructor!
This should work:
```julia
import CLIMAParameters as CP
import SurfaceFluxes.Parameters as SFP
import SurfaceFluxes.UniversalFunctions as UF  # For Businger Params
import Thermodynamics.Parameters as TP

FT = Float64
sf_params = SFP.SurfaceFluxesParameters(FT, UF.BusingerParams)
thermo_params = TP.ThermodynamicsParameters(FT)

# Or with toml dict
toml_dict = CP.create_toml_dict(FT)
sf_params = SFP.SurfaceFluxesParameters(toml_dict, UF.BusingerParams)
thermo_params = TP.ThermodynamicsParameters(toml_dict)
```